pandas/core/accessor.py

```diff
@@ -293,6 +293,66 @@ def register_series_accessor(name):
 
 @doc(_register_accessor, klass="Index")
```

Comment:
I think removing this should fix the doc build

pre-commit is failing, you can run it locally before comitting

Reply:
@mroeschke please see first comment